kenneth051/mydiary-react
# Users should logout

**What does this PR do**
Enables logged in users to log out
**Description of the task being completed**
Currently, our application does not support logging out. This PR enables logged in users to log out of our application whenever they are done using it at the moment. This PR protects some endpoints to only be accessed by logged in users only.

**How should this be manually tested?**
Clone the repository, checkout into this branch
- Run `npm install` to install all dependencies
- Run `npm start`
- register or login then start writing entries.
-Then logout.
-After logging out, go to the create entries endpoint to see if you can access it 
      `localhost:3000/createEntry`

**Relevant Pivotal Tracker Stories**

[#162336575](https://www.pivotaltracker.com/n/projects/2228292/stories/162336575)

#### Any background context you want to add?
N/A

**Screen shots**
 
![screen shot 2018-12-01 at 19 34 19](https://user-images.githubusercontent.com/40117122/49330468-39e9db80-f5a0-11e8-8c90-48bb3491fac6.png)
